scripts/qa_crowbarsetup.sh

```diff
@@ -3296,6 +3301,32 @@ function oncontroller_manila_generic_driver_setup()
         "echo \"ERROR: manila service VM not listening on ssh port. manila tests will fail!\""
 }
 
+function oncontroller_magnum_service_setup ()
+{
+    #TODO: Replace this Fedora image with a suitable SLES image when available
+    local service_image_name=fedora-23-atomic-7.qcow2
+    local service_image_url=https://fedorapeople.org/groups/magnum/$service_image_name
```

Comment:
I added the fedora image for now under http://clouddata.cloud.suse.de/images/other/magnum-service-image.qcow2 (which is a symlink to the fedora-23-atomic-7.qcow2 image). @mmnelemane can you use that link please?